### ISSUE TITLE
Add space between figure number and figure caption on Text Spacing and other SCs

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#1458]
Somewhat of an ironic glitch, given the SC.
There is either inadequate or no space between the Figure number and the figure caption in all illustrations on the Understanding page for Text Spacing. The following image shows figures 1 and 2 exhibiting spacing problem on Chrome (on Mac). Due to thinness of the numeral 1, the issue is not quite as obvious there.
![image](https://user-images.githubusercontent.com/14298245/93668766-a4089480-fa43-11ea-87af-56baf40b5cef.png)

The following image shows figures 2 and 3 exhibiting spacing problem on Safari (on Mac)
<img width="439" alt="image" src="https://user-images.githubusercontent.com/14298245/93668783-c8647100-fa43-11ea-9852-99e6993c809c.png">

Note that this is not isolated to this page. ON the Non-text-contrast page, it is not a problem for figures 3 and 4, but figures 1 and 2 both look like Figure 1A and 2A, and the issue returns at figure 5 and again at figure 9.
![image](https://user-images.githubusercontent.com/14298245/93668877-7f60ec80-fa44-11ea-8a7a-23b585ec8ec0.png)

![image](https://user-images.githubusercontent.com/14298245/93668864-6821ff00-fa44-11ea-89a5-7a85790ac06e.png)

Looking at the source, this seems to be a compilation issue, as the figure number  is generated, not part of the actual code
```
<figure id="figure-text-cut">
            <figcaption>Vertical text cut off  is a failure.</figcaption>
            <img src="https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/images/b/ba/Spacing_cutoff_fail_vertical.png" alt="Heading text truncated vertically." width="420" height="190" />         
         </figure>
```
I thought I'd previously flagged this, but couldn't see any open issue, so have created a new one.

**Comment by patrickhlauke** [comment]
I think https://github.com/w3c/wcag/pull/1459 should fix this (though not had the local dev environment set up to try it)
